ext/hal/st/stm32cube/stm32f1xx/soc/stm32f1xx.h

```diff
@@ -127,33 +127,33 @@
   * @{

```

Comment:
This was an initial approach that I had taken before @erwango  suggested another. This change should have been squashed with the SOC commit, but got here instead. Moved this change into the SoC commit so it doesn't show up in change log anymore.